services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankealueService.kt

```diff
@@ -13,6 +13,8 @@ import org.geojson.Feature
 import org.geojson.FeatureCollection
 import org.springframework.stereotype.Service
 
+const val HANKEALUE_DEFAULT_NAME = "Hankealue"
```

Comment:
Should `HANKEALUE_DEFAULT_NAME` be in Constants.kt?

Reply:
It isn't used anywhere else, so I moved it to a private value in the companion object.